### ISSUE TITLE
Radiant Player interface has white box over control panel

### ISSUE THREAD

**Issue by tmarkovich** [radiant-player/radiant-player-mac#365]
Not sure what's going wrong. This is a fresh binary and when I start it up, I get the following interface. Any suggestions?

![screen shot 2015-10-04 at 9 57 23 am](https://cloud.githubusercontent.com/assets/339699/10268183/dc608fae-6a7e-11e5-9007-84c2ba689e26.jpg)

**Comment by jacobwgillespie** [comment]
This is a duplicate of #351 - the fix has been merged into master and we're waiting on a 1.4.1 release.  If you need this fixed ASAP, there's some links to temporary builds at #351.

**Comment by jadengore** [comment]
@tmarkovich 1.4.1 binary is out, check for updates. See if that works so we can get this issue closed.

**Comment by tmarkovich** [comment]
Fixed.
